pkg/neorpc/result/version_test.go

```diff
@@ -135,6 +143,10 @@ func TestVersion_MarshalUnmarshalJSON(t *testing.T) {
 			expected := new(Version)
 			*expected = *v
 			expected.UserAgent = "/Neo:3.1.0/"
+			// we set SaveInvocations and KeepOnlyLatestState to false
+			// because there are none in SharpNode.
```

Comment:
New line comment should start with the capital letter and end with a period. Inline comment (e.g. `var a = 3 // some comment.`) may start with lowercase, we have both types of usage in the repo.